js/scriptGame.js

```diff
@@ -0,0 +1,130 @@
+let isInventoryOpen = false;
+
+document.getElementsByClassName("inventorySlot slotArrow")[0].onclick = function() {
+    if(!isInventoryOpen){
```

Comment:
```suggestion
    if (!isInventoryOpen) {
```
Fix formating like this everywhere please